spec/ddtrace/contrib/rails/support/rails3.rb

```diff
@@ -43,7 +43,7 @@ def inherited(base)
       config.action_view.javascript_expansions = {}
       config.action_view.stylesheet_expansions = {}
 
-      config.middleware.delete ActionDispatch::DebugExceptions if Rails.version >= '3.2.22.5'
+      # config.middleware.delete ActionDispatch::DebugExceptions if Rails.version >= '3.2.22.5'
       instance_eval(&during_init)
     end
```

Comment:
There's quite a few commented-out lines in this PR -- is this intended? I would suggest perhaps leaving a comment explaining why, if that's the case.

Reply:
Nope, I left commented out code, cleaning it up now.